sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2358,6 +2358,7 @@ object GpuOverrides extends Logging {
       }),
     expr[StringSplit](
        "Splits `str` around occurrences that match `regex`",
+      // Java's split API produces different behaviors than cudf when splitting with empty pattern
```

Comment:
So how is this being handled in Plugin ? Don't we need any change to take care of this case ?

Reply:
We haven't discussed how to handle this in the Plugin. I will follow up with this point. Thank you!

Reply:
Better to file an issue to track this follow-up.

Reply:
@firestarman Thank you! I filed issue: https://github.com/NVIDIA/spark-rapids/issues/7785 to track this (also updated in description above).